modules/nw-operator-cr.adoc

```diff
@@ -120,10 +120,11 @@ configuration.
 are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
 `NetworkPolicy`.
 
-<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This
-value is normally configured automatically, but if the nodes in your cluster do
-not all use the same MTU, then you must set this explicitly to 50 less than the
-smallest node MTU value.
+<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This is detected automatically based on the MTU of the primary network interface. You do not normally need to override the detected MTU.
++
+If the auto-detected value is not what you expected it to be, confirm that the MTU that is set on your nodes is correct. You cannot use this value to change the MTU value on the underlying nodes.
++
+If your cluster requires different MTU values for different nodes, then you must set this value to `50` less than the lowest MTU value in your cluster. For example, if the smallest MTU in your cluster is `900`, then you must set this value to `850`.
```

Comment:
900 is kind of a silly value for MTU though :slightly_smiling_face:... A good real-world example is:

```suggestion
If your cluster requires different MTU values for different nodes, then you must set this value to `50` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, then you must set this value to `1450`.
```

and likewise with the ovn-kube example below but ending up with `1400` instead of `1450`